lib/rails_admin.rb

```diff
@@ -45,7 +45,14 @@ def self.yaml_load(yaml)
   rescue LoadError
     if YAML.respond_to?(:safe_load)
       def self.yaml_load(yaml)
-        YAML.safe_load(yaml)
+        # https://github.com/rails/rails/blob/v6.1.7.6/activerecord/lib/active_record/coders/yaml_column.rb#L50-L56
+        if ActiveRecord::Base.use_yaml_unsafe_load
+          YAML.unsafe_load(yaml)
+        elsif YAML.method(:safe_load).parameters.include?([:key, :permitted_classes])
+          YAML.safe_load(yaml, permitted_classes: ActiveRecord::Base.yaml_column_permitted_classes, aliases: true)
+        else
+          YAML.safe_load(yaml, ActiveRecord::Base.yaml_column_permitted_classes, [], true)
+        end
```

Comment:
This will not affect TE `master` on 3.0.4 because we still use `SafeYAML` and this part of the change will only be used once the `SafeYAML` gem is removed.

The `else` at line 53 will also handle Psych 3.3.2 when it was still using positional arguments instead of keyword arguments. https://github.com/ruby/psych/compare/v3.3.2...v4.0.0#diff-659eac8589abc82c9a0ab3699e4e4be4774d9c09c6c9934af5d9dae0d264439cL330